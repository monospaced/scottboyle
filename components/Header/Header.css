@import url('../../theme/vars.css');

.Header {
  margin: 1.5rem 0;
}

@media (min-width: 639px) {
  .Header {
    margin: 3rem 0 0;
  }
}
