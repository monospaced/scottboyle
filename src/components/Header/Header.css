@import url("../../theme/vars.css");

.Header {
  margin-top: 1.5rem;
}

@media (--larger-viewport) {
  .Header {
    float: left;
    margin-top: 2.25rem;
    text-align: right;
    width: 12.5rem;
  }
}
