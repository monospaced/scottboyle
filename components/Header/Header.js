import React from "react";
import "./Header.css";
import Logotype from "../Logotype/Logotype";

const Header = ({ title, subtitle }) => {
  return (
    <header className="Header" role="banner">
      <Logotype title={title} subtitle={subtitle} />
    </header>
  );
};

export default Header;
