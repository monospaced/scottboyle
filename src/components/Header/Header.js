import React from "react";

import "./Header.css";
import Logotype from "../Logotype/Logotype";

const Header = ({ subtitle, title }) => {
  return (
    <header className="Header" role="banner">
      <Logotype subtitle={subtitle} title={title} />
    </header>
  );
};

export default Header;
