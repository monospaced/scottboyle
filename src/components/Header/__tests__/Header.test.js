import React from "react";
import { shallow } from "enzyme";

import Header from "../Header.js";

describe("Header component", () => {
  it("should render correctly", () => {
    const component = shallow(<Header />);
    expect(component).toMatchSnapshot();
  });
});
