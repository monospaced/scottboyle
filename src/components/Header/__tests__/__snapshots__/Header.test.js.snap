// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render correctly 1`] = `
<header
  className="Header"
  role="banner"
>
  <Logotype />
</header>
`;
